spyder/utils/stylesheet.py

```diff
@@ -486,11 +503,8 @@ def set_stylesheet(self):
             )
 
 
-class HorizontalDockTabBarStyleSheet(BaseDockTabBarStyleSheet):
-    """
-    This implements the design for dockwidget tabs discussed on issue
-    spyder-ide/ux-improvements#4.
-    """
+class SpecialTabBarStyleSheet(BaseDockTabBarStyleSheet):
+    """Style for special horizontal tabbars."""
```

Comment:
Maybe mentioning here that this style is used for the Preferences pages could be worthy?